### DISCUSSION TITLE
Update Grafana and Influx

### DISCUSSION THREAD

**Discussion by s4en** [tteck/Proxmox#1445] [Help Needed] [answered]
I always update the containers with the command apt update && apt upgrade -y.
But now I get the following error message with Grafana and Influx:

> W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://repos.influxdata.com/debian bullseye InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY XXXXXXXXXXXXXXXXX
W: Failed to fetch https://repos.influxdata.com/debian/dists/bullseye/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY XXXXXXXXXXXXXXXXXXXXX
W: Some index files failed to download. They have been ignored, or old ones used instead.

**Comment by tteck** [comment]
To correct the issue, you can execute the install command within the LXC console (needed only once).

**Reply by s4en** [reply]
Thanks for the answer but i running in problems:
Grafana works but in Influx i got this:

> ✗ No InfluxDB Installation Found!

But i can start influx and it works

**Reply by tteck** [reply, marked as answer]
Run the following commands in the InfluxDB LXC console
```
wget -q https://repos.influxdata.com/influxdata-archive_compat.key
cat influxdata-archive_compat.key | gpg --dearmor | tee /etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg > /dev/null
echo 'deb [signed-by=/etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg] https://repos.influxdata.com/debian stable main' | tee /etc/apt/sources.list.d/influxdata.list
rm -f /etc/apt/trusted.gpg.d/influxdb.gpg
apt-get update
apt-get -y upgrade
```

**Reply by s4en** [reply]
Thanks, that did it.
